core/templates/services/UpgradedServices.ts

```diff
@@ -296,6 +296,8 @@ import { PlaythroughIssuesBackendApiService } from
   'services/playthrough-issues-backend-api.service';
 import { PlaythroughObjectFactory } from
   'domain/statistics/PlaythroughObjectFactory';
+import { PlaythroughBackendApiService } from
```

Comment:
Please add this in sorted order (as per import name)

Reply:
Done.